java/quarkus/src/main/java/org/acme/BenchmarkResource.java

```diff
@@ -0,0 +1,33 @@
+package org.acme;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+@Path("/")
+public class BenchmarkResource {
+
+    @GET
+    @Path("/")
+    @Produces(MediaType.TEXT_PLAIN)
+    public String root() {
+        return "";
+    }
+
+    @GET
+    @Path("/user/{id}")
+    @Produces(MediaType.TEXT_PLAIN)
+    public String userId(@PathParam("id") String id) {
```

Comment:
If you use `RESTEasy Reactive` (see comment above), you don't need `@PathParam` at all

Reply:
Nice!